src/components/Layout/index.tsx

```diff
@@ -72,21 +71,18 @@ const Layout = ({ children }: LayoutProps) => {
           }}
         />
         <Link to="/" css={theme.typography.styles.title}>
-          Gatsby Starter Vadyan
+          Der Bruden
         </Link>
         <Nav>
-          <StyledLink to="/" activeClassName="active">
-            Quick start
-          </StyledLink>
-          <StyledLink to="/showcase/" activeClassName="active">
-            Showcase
+          <StyledLink to="https://fantasy.espn.com/football/league?leagueId=794521" activeClassName="active">
```

Comment:
This should technically be a `<a>` since it's an external link and I could set to open in a new tab. For another day.